vulkaninfo/vulkaninfo.h

```diff
@@ -353,13 +564,17 @@ struct AppInstance {
     ANativeWindow *window;
 #endif
     AppInstance() {
-        PFN_vkEnumerateInstanceVersion enumerate_instance_version =
-            reinterpret_cast<PFN_vkEnumerateInstanceVersion>(vkGetInstanceProcAddr(nullptr, "vkEnumerateInstanceVersion"));
+        if (dll.initialize() != VK_SUCCESS) {
+            fprintf(stderr, "Failed to initialize: Vulkan loader is not installed, not found or failed to load.\n");
+            WAIT_FOR_CONSOLE_DESTROY;
+            exit(1);
```

Comment:
ERR_EXIT() for consistency with below?

Reply:
ERR_EXIT doesn't work here, as this isn't a Vulkan function call failing.

Reply:
ah, didn't realize it was Vulkan-specific error reporting